### DISCUSSION TITLE
Please define contributions

### DISCUSSION THREAD

**Discussion by achintya10** [boris-kz/CogAlg#76] [General]
Hi Boris

Please tell me how do you define a contribution in your repository? That will help me contribute correctly and I will be able to show my scripting capabality.

Achintya

**Comment by boris-kz** [comment]
Anything that improves the project, in whatever way. The core of the project is the algorithm to maximize predictive value of representations, as defined in the intro. All of the repo is specifying what that means.

**Comment by achintya10** [comment]
Thanks Boris for your quick and specific reply.

**Reply by boris-kz** [reply]
It's not terribly specific :). For the specifics of what we currently work on, please see my daily commits.
